components/Tabs/src/TabBarView/private/MDCTabBarViewItemView.m

```diff
@@ -150,6 +182,29 @@ - (void)layoutSubviewsImageOnly {
       CGPointMake(CGRectGetMidX(self.contentView.bounds), CGRectGetMidY(self.contentView.bounds));
 }
 
+- (CGRect)contentFrameForTitleLabelInTextAndImageLayout {
+  CGRect contentFrame = UIEdgeInsetsInsetRect(self.bounds, kEdgeInsetsTextAndImage);
+
+  CGSize contentSize = CGSizeMake(CGRectGetWidth(contentFrame), CGRectGetHeight(contentFrame));
+  CGSize labelSingleLineSize = self.titleLabel.intrinsicContentSize;
+  CGSize availableIconSize = CGSizeMake(
+      contentSize.width, contentSize.height - kImageTitlePadding - labelSingleLineSize.height);
```

Comment:
Here and below its confusing for me to see (x - y - z) I think it would be cleaner if it was (x - (y + z)).

Reply:
Done throughout.